views/layout.html

```diff
@@ -0,0 +1,74 @@
+{% extends "govuk/template.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"        import govukBackLink %}
+{% from "govuk/components/button/macro.njk"           import govukButton %}
+{% from "govuk/components/details/macro.njk"          import govukDetails %}
+{% from "govuk/components/error-summary/macro.njk"    import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk"            import govukInput %}
+{% from "govuk/components/phase-banner/macro.njk"     import govukPhaseBanner %}
+{% from "govuk/components/table/macro.njk"            import govukTable %}
+{% from "govuk/components/textarea/macro.njk"         import govukTextarea %}
+{% from "govuk/components/summary-list/macro.njk"     import govukSummaryList %}
+{% from "govuk/components/radios/macro.njk"           import govukRadios %}
+{% from "govuk/components/cookie-banner/macro.njk"    import govukCookieBanner %}
+{% from "govuk/components/footer/macro.njk"           import govukFooter %}
+{% from "govuk/components/header/macro.njk"           import govukHeader %}
+
+{% block head %}
+
+  <!--[if !IE 8]><!-->
+  <link rel="stylesheet" type="text/css" media="all" href="//{{CDN_HOST}}/stylesheets/govuk-frontend/v3.11.0/govuk-frontend-3.11.0.min.css" />
+  <!--<![endif]-->
+
+  <link rel="SHORTCUT ICON" href="//{{CDN_HOST}}/images/favicon.ico" />
+  <link rel="icon" href="//{{CDN_HOST}}/images/favicon.ico" type="image/x-icon" />
+
+  <script src="https://code.jquery.com/jquery-3.4.1.js"
+          integrity="sha256-WpOohJOqMqqyKL9FccASB9O0KwACQJpFTUBLTYOVvVU="
+          crossorigin="anonymous">
+  </script>
+
+{% endblock %}
+
+{% block header %}
+  {{ govukHeader({
+    homepageUrl: "/overseas-entities-web",
```

Comment:
noticed on confirmation-statement-web we have homepage set to www.gov.uk. I think it was raised as a bug previously, here's the PR that changed it - https://github.com/companieshouse/confirmation-statement-web/commit/68b6f76e47a7b42dea4038f0440f75ddf25fb430

Reply:
Thanks Mark, well spotted. Done